libs/interpret-text/src/lib/TextExplanationDashboard/TextExplanationDashboard.tsx

```diff
@@ -0,0 +1,160 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { localization } from "@responsible-ai/localization";
+import {
+  ChoiceGroup,
+  IChoiceGroupOption,
+  Slider
+} from "office-ui-fabric-react";
+import React from "react";
+
+import { RadioKeys, Utils } from "./CommonUtils";
+import { BarChart } from "./Control/BarChart";
+import { TextHighlighting } from "./Control/TextHightlighting";
+import { ITextExplanationDashboardProps } from "./Interfaces/IExplanationDashboardProps";
+import { textExplanationDashboardStyles } from "./TextExplanationDashboard.styles";
+
+export interface ITextExplanationDashboardState {
+  /*
+   * holds the state of the dashboard
+   */
+  maxK: number;
+  topK: number;
+  radio: string;
+}
+
+const options: IChoiceGroupOption[] = [
+  /*
+   * creates the choices for the radio button
+   */
+  { key: RadioKeys.All, text: localization.InterpretText.Dashboard.allButton },
+  { key: RadioKeys.Pos, text: localization.InterpretText.Dashboard.posButton },
+  { key: RadioKeys.Neg, text: localization.InterpretText.Dashboard.negButton }
+];
+
+export class TextExplanationDashboard extends React.PureComponent<
+  ITextExplanationDashboardProps,
+  ITextExplanationDashboardState
+> {
+  constructor(props: ITextExplanationDashboardProps) {
+    /*
+     * initializes the dashboard with its state
+     */
+    super(props);
+    this.state = {
+      maxK: Math.min(
+        15,
+        Math.ceil(Utils.countNonzeros(this.props.dataSummary.localExplanations))
+      ),
+      radio: RadioKeys.All,
+      topK: Math.ceil(
+        Utils.countNonzeros(this.props.dataSummary.localExplanations) / 2
+      )
+    };
+    this.changeRadioButton = this.changeRadioButton.bind(this);
+  }
+
+  public render() {
+    const classNames = textExplanationDashboardStyles();
+    return (
+      <div className={classNames.explainerDashboard}>
```

Comment:
I think it may be better to change some of these to stacks instead of divs?  any thoughts on this?  maybe in a follow-up PR.

Reply:
Yes, stack is a lot better

Reply:
done!